packages/zoe/test/types.test-d.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file uses .ts syntax to be able to declare types (e.g. of kit.creatorFacet as {})
+ * because "there is no JavaScript syntax for passing a a type argument"
+ * https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html
```

Comment:
I'm confused about the contribution of this file. I thought .ts fles were for type declarations, but this looks like imperative code, though not working code. What is its impact? What types are being declared or implied? Does it run as code during tests, or is it parsed and discarded, or something else?

Reply:
Thanks for the explanations.  Would it make sense to add any of this as a comment to the file?

Reply:
It would have to be duplicated for each file. I've put it in the wiki: https://github.com/Agoric/agoric-sdk/wiki/Static-types-testing